src/examples/hotgym/HelloSPTP.cpp

```diff
@@ -115,33 +115,32 @@ Real64 BenchmarkHotgym::run(UInt EPOCHS, bool useSPlocal, bool useSPglobal, bool
 
     //Encode
     tEnc.start();
-    enc.encodeIntoArray(r, input.data());
+    enc.encodeIntoArray(r, input.getSparse().data());
```

Comment:
is this a correct use? I want to easily write to SDR's buffer.

Reply:
No, im pretty sure that you want to write to the dense formatted data buffer here.  Replace with getDense().data() and you should be good.  

Also this probably wont compile because the SDRs dense data is 8bit int and the method encodeIntoArray expects ???

Reply:
good catch! thanks
And it does seem to compile, though